aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -1291,7 +1291,17 @@ public void confirmSignIn(final String signInChallengeResponse,
                               final Callback<SignInResult> callback) {
 
         final InternalCallback internalCallback = new InternalCallback<SignInResult>(callback);
-        internalCallback.async(_confirmSignIn(signInChallengeResponse, clientMetadata, internalCallback));
+        internalCallback.async(_confirmSignIn(signInChallengeResponse, clientMetadata, Collections.<String, String>emptyMap(), internalCallback));
```

Comment:
nit: remove redundancy

```suggestion
        confirmSignIn(signInChallengeResponse, clientMetadata, Collections.<String, String>emptyMap(), callback);
```

Reply:
Would prefer to leave this since in order to keep the changes on this PR minimal.